packages/firestore/src/core/sync_engine.ts

```diff
@@ -860,10 +860,12 @@ export class SyncEngine implements RemoteSyncer {
     );
   }
 
-  async handleCredentialChange(user: User): Promise<void> {
+  async handleUserChange(user: User): Promise<void> {
```

Comment:
There are still plenty of other references to `handleCredentialChange` and `CredentialChangeListener`. It seems preferable to keep these consistent, no? That way you can search for credentialchange and find all the places we do something about this.

Reply:
I renamed the RemoteStore and SyncEngine method to `handleCredentialChange`. LocalStore and SharedClientState still use `handleUserChange` since SyncEngine "swallows" no-op user changes.